examples/plutus-nft/src/Cardano/PlutusExample/NFT.hs

```diff
@@ -65,8 +65,8 @@ type NFTSchema =
 
 mintNft :: TokenName -> Contract (Last Text) NFTSchema Text ()
 mintNft tn = do
-  pkh <- Contract.ownPubKeyHash
-  utxos <- utxosAt (pubKeyHashAddress pkh)
+  pkh <- Contract.ownPaymentPubKeyHash
```

Comment:
Is there not a way to get own staking address?

Reply:
There don't seem to be a way to do that: https://github.com/input-output-hk/plutus-apps/blob/main/plutus-contract/src/Plutus/Contract.hs#L66

Reply:
Fair enough